pkg/errs/errno.go

```diff
@@ -217,6 +217,7 @@ var (
 	ErrServerNotStarted       = errors.Normalize("server not started", errors.RFCCodeText("PD:server:ErrServerNotStarted"))
 	ErrRateLimitExceeded      = errors.Normalize("rate limit exceeded", errors.RFCCodeText("PD:server:ErrRateLimitExceeded"))
 	ErrLeaderFrequentlyChange = errors.Normalize("leader %s frequently changed, leader-key is [%s]", errors.RFCCodeText("PD:server:ErrLeaderFrequentlyChange"))
+	ErrMissingStoreId         = errors.Normalize("missing store id", errors.RFCCodeText("PD:server:ErrMissingStoreId"))
```

Comment:
maybe can include miss store id?

Reply:
Sorry, I don't quite understand what you mean

Reply:
Oh, just like `store id %d`, you can check the above `ErrLeaderFrequentlyChange` usage